clients/ptranslator/src/test/java/org/hawkular/metrics/clients/ptrans/fullstack/CollectdITest.java

```diff
@@ -264,6 +268,7 @@ private Point collectdLogToPoint(String line) {
             String type = split[split.length - 1];
 
             urlConnection = (HttpURLConnection) new URL(findGaugeDataUrl(metricName)).openConnection();
+            urlConnection.setRequestProperty("tenantId", tenant);
```

Comment:
On 12 May 2015, at 14:06, Thomas Segismont wrote:
> names. So #toString may be more natural. Is that your thinking?

Yes. And can even be omitted, as the implicit call to `toString()` would
then do the right thing already.